### DISCUSSION TITLE
Save an incomplete Issue Form as a draft

### DISCUSSION THREAD

**Discussion by grafst** [community/community#62602] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Would be really nice. I already filled in all the details (software version and so on), but I can't complete the reproduction repository today. So tomorrow I will have to fill in all the details again. 
Would be really nice to have a "save as draft" option.

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by avezina** [comment]
When creating a new issue, sometimes the browser times out or I close the tab by accident. In some cases, the issue contents I typed are lost. 😭 
Time to start over!
It would be great if new issues could auto-save as a draft once your start creating it so changes can be saved to the draft and never get lost!
